client/main/notification.js

```diff
@@ -0,0 +1,94 @@
+
+'use strict';
+
+const _ = require('underscore');
+var $ = require('jquery');
+var Backbone = require('backbone');
+Backbone.$ = $;
+var SilkyView = require('./view');
+
+var Notification = SilkyView.extend({
+    className: "notification",
+    events : {
+        'click .silky-notification-link'    : '_linkClicked'
+    },
+    initialize: function() {
+        this.model.on("change", this.render, this);
+        this.model.on("change:index", this.movedown, this);
+        this.bottom = (10*(this.model.attributes.index+1)+50*this.model.attributes.index);
+        this.render();
+    },
+    render: function() {
+
+        this.$el.addClass("silky-notification");
```

Comment:
so `render()` is designed to be called multiple times, does this add multiple classes to $el?

it doesn't matter, but you could shift this line into `initialize()`